core/testing/src/commonMain/kotlin/io/github/droidkaigi/confsched/testing/robot/sessions/TimetableItemDetailScreenRobot.kt

```diff
@@ -0,0 +1,287 @@
+package io.github.droidkaigi.confsched.testing.robot.sessions
+
+import androidx.compose.ui.test.ComposeUiTest
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertTextContains
+import androidx.compose.ui.test.assertTextEquals
+import androidx.compose.ui.test.hasTestTag
+import androidx.compose.ui.test.onAllNodesWithTag
+import androidx.compose.ui.test.onChildren
+import androidx.compose.ui.test.onFirst
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onRoot
+import androidx.compose.ui.test.performScrollToIndex
+import androidx.compose.ui.test.performScrollToNode
+import androidx.compose.ui.test.performTouchInput
+import androidx.compose.ui.test.swipeUp
+import dev.zacsweers.metro.Inject
+import io.github.droidkaigi.confsched.data.sessions.FakeSessionsApiClient
+import io.github.droidkaigi.confsched.model.sessions.TimetableItemId
+import io.github.droidkaigi.confsched.sessions.TimetableItemDetailScreenContext
+import io.github.droidkaigi.confsched.sessions.TimetableItemDetailScreenLazyColumnTestTag
+import io.github.droidkaigi.confsched.sessions.TimetableItemDetailScreenRoot
+import io.github.droidkaigi.confsched.sessions.components.DescriptionMoreButtonTestTag
+import io.github.droidkaigi.confsched.sessions.components.SummaryCardTextTag
+import io.github.droidkaigi.confsched.sessions.components.TargetAudienceSectionTestTag
+import io.github.droidkaigi.confsched.sessions.components.TimetableItemDetailContentTargetAudienceSectionBottomTestTag
+import io.github.droidkaigi.confsched.sessions.components.TimetableItemDetailHeadlineTestTag
+import io.github.droidkaigi.confsched.sessions.components.TimetableItemDetailMessageRowTestTag
+import io.github.droidkaigi.confsched.sessions.components.TimetableItemDetailMessageRowTextTestTag
+import io.github.droidkaigi.confsched.testing.compose.TestDefaultsProvider
+import io.github.droidkaigi.confsched.testing.robot.core.CaptureScreenRobot
+import io.github.droidkaigi.confsched.testing.robot.core.DefaultCaptureScreenRobot
+import io.github.droidkaigi.confsched.testing.robot.core.DefaultFontScaleRobot
+import io.github.droidkaigi.confsched.testing.robot.core.DefaultWaitRobot
+import io.github.droidkaigi.confsched.testing.robot.core.FontScaleRobot
+import io.github.droidkaigi.confsched.testing.robot.core.WaitRobot
+import kotlinx.coroutines.test.TestDispatcher
+
+@Inject
+class TimetableItemDetailScreenRobot(
+    private val contextFactory: TimetableItemDetailScreenContext.Factory,
```

Comment:
- When referring to last year's content, it is necessary to change the timetableItemId according to the test content, so unlike other robots, we have made contextFactory a dependency.
- https://github.com/DroidKaigi/conference-app-2024/blob/57c38a76beb5b75edc9220833162e1257f40ac06/core/testing/src/main/java/io/github/droidkaigi/confsched/testing/robot/TimetableItemDetailScreenRobot.kt#L48-L68